02/08/2024 11:49:02 +07:00 [Information] - Message: Query Data From Table Category
02/08/2024 11:51:04 +07:00 [Information] - Message: Query Data From Table Category
02/08/2024 11:57:46 +07:00 [Information] - Message: Query Data From Table Category
02/08/2024 12:06:24 +07:00 [Error] - Message: An unhandled exception has occurred while executing the request.
KineMartAPI.Exceptions.UniqueException: CategoryName was already exist
   at KineMartAPI.ServiceImpls.CategoryService.Check(Category category) in C:\Users\ASUS\source\repos\KineMartAPI\KineMartAPI\ServiceImpls\CategoryService.cs:line 83
   at KineMartAPI.ServiceImpls.CategoryService.AddCategory(List`1 categories) in C:\Users\ASUS\source\repos\KineMartAPI\KineMartAPI\ServiceImpls\CategoryService.cs:line 20
   at KineMartAPI.Controllers.CategoryController.Create(List`1 categoryDtos) in C:\Users\ASUS\source\repos\KineMartAPI\KineMartAPI\Controllers\CategoryController.cs:line 30
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(ActionContext actionContext, IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddlewareImpl.<Invoke>g__Awaited|8_0(ExceptionHandlerMiddlewareImpl middleware, HttpContext context, Task task)
02/08/2024 12:06:48 +07:00 [Information] - Message: Query Data From Table Category
